Cypress/cypress/pages/workflow/workflowRecurring.js

```diff
@@ -0,0 +1,21 @@
+///  Validate stats for recurring 
+
+Cypress.Commands.add("validateRecurringStats", () => {
+    cy.wait(1000);

```

Comment:
Instead of waiting, we can check the visibility of statsHeatMap & same for statsBarGraph?

Reply:
Required work from https://github.com/litmuschaos/litmus/pull/3322

Reply:
Can't we use just .should("be.visible") for statsGraphn &  HeatMap, we don't need to check loader?